### PR TITLE
Added support for Get-Content and Set-Content

### PR DESCRIPTION
**Pull request by jianyunt**

### CODE REVIEW

File: src/Microsoft.PowerShell.SHiPS/Node/ContentWriter.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        /// <summary>
+        /// Moves the current stream position.
+        /// </summary>
+        ///
+        /// <param name="offset"> The offset from the origin to move the position to. </param>
+        /// <param name="origin"> The origin from which the offset is calculated. </param>
+        public void Seek(long offset, SeekOrigin origin)
+        {
+            _writer?.Flush();
+            _stream.Seek(offset, origin);
```
Comment (edyoung) [inline src/Microsoft.PowerShell.SHiPS/Node/ContentWriter.cs]: not sure why writer gets ?. and stream does not

File: docs/PublicAPIsAndMore.md [hunk 2]
```diff
@@ -47,6 +47,12 @@ With that,  in PowerShell, you can pass in a node name argument as follows.
 
     Defines Get-ChildItem dynamic parameters for `Get-ChildItem`
 
+- [object] SetContent([string]$content, [string]$path)
+  Sets the content string text in $content to the path node specified in $path.
+
+- [object] GetContent()
+  Gets the content text from the current node object.
+
```
[the remark this answers is not in the document]
Comment (jianyunt) [reply docs/PublicAPIsAndMore.md]: Good catch! Changed to string.

File: samples/Library/Library.psm1 [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+{
+    hidden [string]$data = "Tchaikovsky's magical ballet tells the story of the doomed love of Prince Siegfried and Princess Odette. 
+    Prince Siegfried goes out hunting one night and chases a group of swans – one of them transforms into a young woman, Odette, who explains 
+    that she and her companions were turned into swans by the evil Baron
+    Von Rothbart."
+
+    SwanLake() : base ($this.GetType())
+    {
+    }
+
```
[the remark this answers is not in the document]
Comment (jianyunt) [reply samples/Library/Library.psm1]: You are absolutely right. We do not need them. I created these SwanLake and BlueDanub classes first and then added MusicLeaf. Now it's clean now.

File: src/Microsoft.PowerShell.SHiPS/Node/ContentReader.cs [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            _writer?.Flush();
+            _reader?.DiscardBufferedData();
+        }
+
+        /// <summary>
+        /// Closes the stream.
+        /// </summary>
+        public void Close()
+        {
+            if (_writer == null && _reader == null) { return; }
```
[the remark this answers is not in the document]
Comment (jianyunt) [reply src/Microsoft.PowerShell.SHiPS/Node/ContentReader.cs]: fixed.thx

File: src/Microsoft.PowerShell.SHiPS/Node/ContentWriter.cs (hunk 1 above)
Comment (jianyunt) [reply src/Microsoft.PowerShell.SHiPS/Node/ContentWriter.cs]: fixed.thx.

**Pull request merged**